docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -550,6 +560,21 @@ The following example uses all three table properties::
   )
   AS SELECT name, comment, regionkey FROM tpch.tiny.nation;
 
+.. _delta-lake-register-table-support:
+
+Register table
+^^^^^^^^^^^^^^
+
+The connector can register table into the metastore with existing transaction logs and data files.
+
+An SQL procedure ``system.register_table`` allows the caller to register an existing delta lake
```

Comment:
We may want to remove `An` according to #15018. Or "The ``register_table`` procedure allows..." is acceptable to me.